asset-registry/src/lib.rs

```diff
@@ -14,7 +14,7 @@ use sp_runtime::{
 	DispatchResult,
 };
 use sp_std::prelude::*;
-use xcm::{v3::prelude::*, VersionedMultiLocation};
+use xcm::{v4::prelude::*, VersionedLocation};
```

Comment:
please keep this v3, otherwise it is a breaking change that requires migration
while we can migrate to v4, let's wait until v4 is battle tested first and there is a clear deprecation plan for v3

Reply:
The SCALE encoding did not change with the new xcm::v4::Location, I added a test to just to be sure https://github.com/open-web3-stack/open-runtime-module-library/pull/977/commits/38f45f2d9d7e652040444630f12679f99e8acbe0

I could add a migration, but seems unnecessary and expensive.

Reply:
The same applies to pallet `unknown-tokens`, it does not require a migration.